packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -103,12 +105,24 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
     setOpen(false);
   }, []);
 
+  const hidePanel = useCallback(() => {
+    setOpen(false);
```

Comment:
Should this be

```suggestion
    setShowResultsPanel(false);
```

?